inputs/worker_inner.json

```diff
@@ -0,0 +1,158 @@
+{
+  "client_version": "{{VERSION}}",
+  "prebuild_plugins": [
+    {
+      "name": "add_filesystem",
+      "args": {
+        "koji_hub": "{{KOJI_HUB}}",
+        "from_task_id": "{{FILESYSTEM_KOJI_TASK_ID}}"
+      }
+    },
+    {
+      "name": "select_platform",
+      "args": {
+        "repository_transform": null
+      },
+      "required": false
+    },
+    {
+      "name": "pull_base_image",
+      "args": {
+        "parent_registry": "{{REGISTRY_URI}}",
+        "parent_registry_insecure": true
+      }
+    },
+    {
+      "args": {
+        "labels": "{{IMPLICIT_LABELS}}"
+      },
+      "name": "add_labels_in_dockerfile"
+    },
+    {
+      "name": "change_from_in_dockerfile"
+    },
+    {
+      "name": "add_help",
+      "required": false
+    },
+    {
+      "name": "add_dockerfile"
+    },
+    {
+      "args": {
+        "command": "{{SOURCES_COMMAND}}"
+      },
+      "name": "distgit_fetch_artefacts"
+    },
+    {
+      "args": {
+        "root": "{{KOJI_ROOT}}",
+        "target": "{{KOJI_TARGET}}",
+        "hub": "{{KOJI_HUB}}"
+      },
+      "name": "koji"
+    },
+    {
+      "args": {
+        "repourls": []
+      },
+      "name": "add_yum_repo_by_url"
+    },
+    {
+      "name": "inject_yum_repo"
+    },
+    {
+      "name": "distribution_scope"
+    }
+  ],
+  "buildstep_plugins": [
+    {
+      "name": "docker_api"
+    }
+  ],
+  "prepublish_plugins": [
+    {
+      "name": "squash"
+    }
+  ],
+  "postbuild_plugins": [
+    {
+      "args": {
+        "image_id": "BUILT_IMAGE_ID"
+      },
+      "name": "all_rpm_packages"
+    },
+    {
+      "name": "tag_by_labels"
+    },
+    {
+      "name": "tag_from_config"
+    },
+    {
+      "name": "tag_and_push",
+      "args": {
+        "registries": {
+          "{{REGISTRY_URI}}": { "insecure": true }
+        }
+      }
+    },
+    {
+      "name": "pulp_init",
+      "args": {},
+      "required": false
+    },
+    {
+      "name": "pulp_push",
+      "args": {
+        "pulp_registry_name": "{{PULP_REGISTRY_NAME}}",
+        "load_exported_image": true,
+        "dockpulp_loglevel": "INFO"
+      }
+    },
+    {
+      "name": "pulp_sync",
+      "args": {
+        "pulp_registry_name": "{{PULP_REGISTRY_NAME}}",
+        "docker_registry": "{{DOCKER_REGISTRY}}",
+        "dockpulp_loglevel": "INFO"
+      }
+    },
+    {
+      "name": "compress",
+      "args": {
+        "load_exported_image": true,
+        "method": "gzip"
+      }
+    },
+    {
+      "name": "pulp_pull",
+      "args": {}
+    },
+    {
+      "name": "koji_upload",
```

Comment:
Should this be koji_promote?

With this pull request, I think we want to be able to create a single worker build and have it complete on its own. When we get to orchestration, later, we can decide whether to start off with supporting only a single worker build, or to support multiple worker builds (and use koji_upload in the worker builds).